code/modules/organs/robolimbs.dm

```diff
@@ -98,6 +98,14 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	icon = 'icons/mob/human_races/cyberlimbs/unbranded/unbranded_unathi.dmi'
 	unavailable_to_build = 1
 
+/datum/robolimb/unbranded_teshari
+	company = "Unbranded - Teshari"
+	species_cannot_use = list(SPECIES_UNATHI, SPECIES_PROMETHEAN, SPECIES_DIONA, SPECIES_HUMAN, SPECIES_VOX, SPECIES_HUMAN_VATBORN, SPECIES_TAJ, SPECIES_SKRELL, SPECIES_ZADDAT)
+	suggested_species = SPECIES_TESHARI
+	desc = "A simple robotic limb with a small, raptor-like design. Seems rather stiff."
+	icon = 'icons/mob/human_races/cyberlimbs/unbranded/unbranded_teshari.dmi'
+	unavailable_to_build = 0
```

Comment:
The other species unbranded are set to 1, is this intentional?

Reply:
it is, that's a fix to it not showing in the prosthetic fab even after using the disk

Reply:
I believe the other two are set to be unavailable to be built because they're identified in the base unbranded limb's datum as alternatives, so the code will nominally use that instead? Would have to ask mechoid for specifics though